sdk/trafficmanager/Azure.ResourceManager.TrafficManager/src/Customization/HeatmapType.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.ResourceManager.TrafficManager.Models
+{
+    /// <summary>
+    /// Renaming Enum8 enum appropriately.
+    /// </summary>
+    [CodeGenModel("Enum8")]
+    public readonly partial struct HeatMapType
```

Comment:
lets do this with a rename in the md instead of this customization file.  An [example can be found here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/src/autorest.md) searching for enum.

looks like this
```
  - from: resources.json
    where: $.definitions.AliasPathMetadata.properties.attributes["x-ms-enum"]
    transform:
      $["name"] = "ResourceTypeAliasPathAttributes";
```

Reply:
Sure, let's do it. But for me reference, what's the benefit of doing it in the md vs class customization?

Reply:
After scratching the surface, I do not think the advice is applicable here. 

The Enum8 gets generated automatically from the following definition (and is not x-ms-enum definition as in the example provided):

"operationId": "HeatMap_Get",
        "description": "Gets latest heatmap for Traffic Manager profile.",
        "parameters": [
....
{
            "name": "heatMapType",
            "in": "path",
            "required": true,
            "type": "string",
            "enum": [
              "default"
            ],
            "description": "The type of HeatMap for the Traffic Manager profile."
          }
...
]